fabio.properties

```diff
@@ -505,6 +506,46 @@
 # metrics.statsd.addr =
 
 
+# metrics.circonus.apiapp configures the API token app to use when
+# submitting metrics to Circonus. See: https://login.circonus.com/user/tokens
+# This is optional when ${metrics.target} is set to "circonus".
+#
+# The default is
+#
+# metrics.circonus.apiapp = "fabio"
```

Comment:
Added a default for CirconusAPIApp to defaults. I reworded the description of the API URL parameter, I'd prefer to keep that default localized in the library so that if it changes there isn't a waterfall of PRs to update.